tests/e2e/specs/dashboard-samples/RecommendedExtensions.spec.ts

```diff
@@ -91,27 +94,39 @@ for (const sample of samples) {
 		});
 
 		test(`Get recommended extensions list from ${extensionsListFileName}`, async function (): Promise<void> {
-			await (await projectAndFileTests.getProjectTreeItem(projectSection, pathToExtensionsListFileName))?.select();
+			// sometimes the Trust Dialog does not appear as expected - as result we can get opened Trust Box dialog. In this case.
+			// we need to perform this case
```

Comment:
did you mean `we need to execute "projectAndFileTests.performManageWorkspaceTrustBox()" method` ?